indexing-service/src/main/java/org/apache/druid/indexing/common/task/CompactionTask.java

```diff
@@ -1523,4 +1523,10 @@ public CompactionTuningConfig withPartitionsSpec(PartitionsSpec partitionsSpec)
       );
     }
   }
+
+  @Override
+  public List<String> getLookupsToLoad()
+  {
+    return Collections.emptyList();
```

Comment:
We can use lookup in an aggregation function so I think this should not be done.

Reply:
I'm lacking context regarding compaction task. The other PR disabled loading lookups for compaction tasks. The original issue (https://github.com/apache/druid/issues/13324#issue-1439073000) also states that task types such as compaction have no use of lookups.

@cryptoe Who would be the right person to confirm this?

cc: @kfaraz @IgorBerman

Reply:
I assumed compactions are not using lookups since they load already stored data and save it as is in bigger files or with some rollup. I don't see how to apply aggregation function in documentation of compaction task https://druid.apache.org/docs/latest/data-management/manual-compaction#compaction-transform-spec, but there might be some case which I'm not aware of.
Anyway, your next PR will permit to configure lookups used per task in config as per my understanding, so I would be able to disable loading lookups for our specific case (simple rollups)

Reply:
As per discussion, I have removed compaction task related changes from this PR. Will take that up as part of next PR.